code/modules/clothing/suits/cloaks.dm

```diff
@@ -66,7 +66,7 @@
 	icon_state = "dragon"
 	desc = "A suit of armour fashioned from the remains of an ash drake. "
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/internals, /obj/item/weapon/resonator, /obj/item/device/mining_scanner, /obj/item/device/t_scanner/adv_mining_scanner, /obj/item/weapon/gun/energy/kinetic_accelerator, /obj/item/weapon/pickaxe, /obj/item/weapon/twohanded/spear)
-	armor = list(melee = 70, bullet = 30, laser = 50, energy = 40, bomb = 70, bio = 60, rad = 50)
+	armor = list(melee = 70, bullet = 30, laser = 50, energy = 40, bomb = 70, bio = 0, rad = 0)
```

Comment:
Decloner inherits the `"energy"` flag from its parent, so, no.

even reagent protection goes off of permeability coefficient. 

I kinda get the idea that someone intended to get rid of one or the other, at some point, but never got around to doing it.